src/main/resources/site/parts/statistics/statistics.es6

```diff
@@ -59,21 +64,45 @@ const renderPart = (req) => {
     statisticsKeyFigure = keyFigurePreview(req, page.data.statisticsKeyFigure)
   }
 
+  if (moment(page.data.lastModified).isAfter(statistic.variants.previousRelease)) {
+    changeDate = moment(page.data.lastModified).format('DD. MMMM YYYY, HH:MM')
+  }
+
+  const modifiedDateComponent = new React4xp('ModifiedDate')
+    .setProps({
+      explanation: modifiedText,
+      className: '',
+      children: changeDate
+    })
+    .setId('modifiedDate')
+    .uniqueId()
+
   const model = {
     title,
     updated,
     nextUpdate,
+    changed,
+    changeDate,
+    modifiedText,
     previousRelease,
     nextRelease,
+    modifiedDateId: modifiedDateComponent.react4xpId,
     statisticsKeyFigure: statisticsKeyFigure ? statisticsKeyFigure.body : null
   }
 
-  const body = render(view, model)
-  return {
-    body,
+  let body = render(view, model)
+  body = modifiedDateComponent.renderBody({
+    body
+  })
+  pageContributions = modifiedDateComponent.renderPageContributions({
```

Comment:
Lint: _'error  'pageContributions' is never reassigned. Use 'const' instead  prefer-cons'_

You can just delete 'let pageContributions' and use const pageContributions here

Reply:
fixed